go.mod

```diff
@@ -163,5 +163,7 @@ replace (
 	// TODO: remove when gravity update dependencies
 	github.com/peggyjv/gravity-bridge/module/v2 => github.com/crypto-org-chain/gravity-bridge/module/v2 v2.0.0-20220509015341-a016e2b04866
 
+	github.com/tharsis/ethermint => github.com/yihuang/ethermint v0.6.1-0.20220524072845-48dfc975f7ba
```

Comment:
https://github.com/tharsis/ethermint/pull/1062

Reply:
I have the same question in https://github.com/crypto-org-chain/cronos/issues/504
Why the `main` branch is still using ethermint `v0.6.1`?

Reply:
It’s not, what matters is the commit hash, the tag name is inferred by golang compiler from the forked repo